zipline/data/minute_bars.py

```diff
@@ -552,6 +552,10 @@ def _open_minute_file(self, field, sid):
 
         return carray
 
+    def get_ctable(self, sid):
+        ctable_path = os.path.join(self._rootdir, _sid_subdir_path(sid))
```

Comment:
This looks like it's replicating the behavior of `_ensure_ctable`, except without the guaranteed initialization.  When would we want to call this without guaranteeing that the table already exists?